### DISCUSSION TITLE
Disabled Veterans with memory issues and forgotten passwords

### DISCUSSION THREAD

**Discussion by Ryanbry123** [community/community#144845] [Accessibility] [answered]
### Select Topic Area

General

### GitHub Feature Area

Mobile

### Body

How can a disabled veteran from concussion and memory loss relocate lost email accounts and passwords to vital information?  Without the ptsd aka frustration agitation?  An app out there that allows a person to safely enter their ssn and pull up lost info especially passwords

**Comment by Ryanbry123** [comment]
Help

**Comment by cehfisher** [comment, marked as answer]
Hello @Ryanbry123 - sorry to hear you are having difficulties logging in to your account. The best place to go for answers to account-specific questions and direct assistance is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket and the support team will follow-up with next steps. Thank you!
